### ISSUE TITLE
Placeholder text in all Inputs and Text Area does not pass AA

### ISSUE THREAD

**Issue by ashetland** [Esri/calcite-design-system#10361]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Summary

Placeholder text should pass AA standards in both light and dark modes.

### Actual Behavior

Our current placeholder text is #757575 in both light and dark modes. This passes against `foreground.1` in light mode but not in dark mode. 

### Expected Behavior

Placeholder text should be assigned `color.text.3` which passes AA in both color modes.

### Reproduction Sample

https://codepen.io/ashetland/pen/bGPXOVa?editors=1000

### Reproduction Steps

See the placeholder color not changing in dark mode. 

### Reproduction Version

v2.12

### Relevant Info

Components affected:

- [ ] Input
- [ ] Input Number
- [ ] Input Text
- [ ] Input Date Picker
- [ ] Input Time Zone/Combobox
- [ ] Text Area

### Priority impact

impact - p2 - want for an upcoming milestone

### Calcite package

- [X] @esri/calcite-components
- [ ] @esri/calcite-components-angular
- [ ] @esri/calcite-components-react
- [X] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

Calcite (design)

**monday.com sync:** #8881237511

**Comment by ashetland** [comment]
I took a look at this again and it seems our placeholder text color is inconsistently implemented. This is an updated v3.1 [codepen](https://codepen.io/ashetland/pen/bGPXOVa) that includes all the components mentioned above. Design resolution is still to update placeholder text to `color.text.3`.

**Comment by DitwanP** [comment]
🍠 Verified on `3.2.0-next.23`
